### ISSUE TITLE
Siri commands not working. Possibly due to the command needing "title:" in it?

### ISSUE THREAD

**Issue by abhiiously** [darkalfx/requestrr#120]
I added the get contents of to include "/request movie title:" but that doesnt seem to do anything. Any idea?

**Comment by mrzoops** [comment]
Any ETA on getting the siri shortcuts working again? I tried playing with it and changing the text output but can't get it to recognize it as a command.

**Comment by iamhenry** [comment]
would love to see this working again. any updates here?

thx for all the work you've done!

**Comment by brkr1** [comment]
> would love to see this working again. any updates here?
> 
> thx for all the work you've done!

Have you had any success on that matter?

**Comment by iamhenry** [comment]
> > would love to see this working again. any updates here?
> > thx for all the work you've done!
> 
> Have you had any success on that matter?

nope, i think the siri shortcut format has to be recreated but i dont know how using a bit of code fetching discord api
